webhook_to_fedora_messaging/config/defaults.py

```diff
@@ -0,0 +1,40 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+class Defaults:
+    DEBUG = True
+    PERMANENT_SESSION_LIFETIME = 604800
+    SESSION_COOKIE_NAME = "user"
```

Comment:
Why this name change? Is it going to contain only user-related stuff?

Reply:
Yes.